components/ExpandableText.tsx

```diff
@@ -34,26 +34,39 @@ const ExpandableText: React.FunctionComponent<ExpandableTextProps> = (
         setTruncated(!truncated);
     };
 
+    // Disable toggler for now
+    let options: Array<JSX.Element> = [];
+    let tokens = props.fullText.split('#');
+    for (let token of tokens) {
+        if (token.length > 0) {
+            options.push(
+                <span>
+                    - {token.toLowerCase()}
+                    <br />
+                </span>
+            );
+        }
+    }
```

Comment:
I would use filter and map combination here. Something like

```suggestion
    const options = tokens.filter(token => token.length > 0).map(token => 
        <span>
            - {token.toLowerCase()}
            <br />
        </span>
    );
```

Reply:
fancy :-)

Reply:
@onursumer that's some nice code :-)

Added here:  https://github.com/ncihtan/htan-portal/pull/440/commits/280b2c4723f0b80610574f04a4152bd9c3ee9bb8